### DISCUSSION TITLE
APP check box not found in iDrive under phone settings

### DISCUSSION THREAD

**Discussion by lawyeradam** [BimmerGestalt/AAIdrive#672] [Q&A]
Hello all,  I do not have APPs for this phone (OnePlus 9 Pro) in the iDrive.  I have it for two other phones - S21 and LG V30+.  Any ideas?  I've uninstalled, reinstalled, unpaired, repaired, updated BMW Connected Drive via Options, called Genius (didn't help).  Is it possible BMW just doesn't like OnePlus devices?  Bluetooth streams just fine.  

![image](https://user-images.githubusercontent.com/106994695/218873465-5056d78e-b524-4493-9299-ded806f82452.png)
This is my V30 and it has the APPS selection.

![image](https://user-images.githubusercontent.com/106994695/218874457-11a19840-474b-4197-9d3d-4c3ac1de284b.png)
OnePlus 9 Pro - Missing Apps selection

**Comment by hufman** [comment]
When you paired this device in the car, did you pair it for Apps or just for Music? Try unpairing the phone and going through the car's Pairing screen but selecting Apps there.

**Reply by lawyeradam** [reply]
Hi and Thanks for the support.  I just tried this several times and no luck.  The car and phone will make the bluetooth audio / call connection but the idrive says Connecting.... Start app if necessary.  and the circle keeps spinning and spinning.  Then no Apps Box for this phone.  Any additional ideas?  Also, Connected Drive was logged in and active and the AAIdrive app was running.

**Reply by galenzhao** [reply]
Try to unpair all phones,
And re-pair your phone while opening the mybmw app.

**Comment by TheJapalian** [comment]
I am having the same issue with my Pixel 8 pro. I know apps worked with my pixel 7 and continue to work with my wife 's s22. I've tried un-pairing and re-pairing but Apps never show up with my pixel 8. Any ideas?

**Reply by hufman** [reply]
Unfortunately I don't know any further steps to try, beyond un-pairing and re-pairing while MyBMW/Mini is open. Please try asking your car dealer about what to try next.
